SSA/Projects/Tensor2D/Tensor2D.lean

```diff
@@ -205,14 +168,14 @@ theorem map_fill_2d
     (sz₀ sz₁ ix₀ ix₁: Index) 
     (i : Int):
   TSSA.eval
-  (e := e) (Op := Op) (β := BaseType) [dsl_bb2|
+  (e := e) (Op := Op) (β := ) [dsl_bb2|
     return op:fill2d (op:constInt(i) (), 
       op:extract2d (
           ((op:constIx(sz₀) () , op:constIx(sz₁) ()),
            (op:constIx(ix₀) (), op:constIx(ix₁) ())),
            op:constTensor(t) ()))
   ] =
-  TSSA.eval (e := e) (Op := Op) (β := BaseType) [dsl_bb2|
+  TSSA.eval (e := e) (Op := Op) (β := ) [dsl_bb2|
```

Comment:
```suggestion
  TSSA.eval (e := e) (Op := Op) [dsl_bb2|
```
I'm surprised this is legal syntax, but we probably want to remove the unspecified argument completely (also at line 171 above).

Reply:
You forgot the one on line 171, so I took the liberty of removing it